libs/doc-components/src/components/docCanvas/docCanvas.tsx

```diff
@@ -18,13 +18,15 @@ interface SourceType {
 interface DocCanvasProps {
   activeTab: string
   children: React.ReactNode
+  display: string
```

Comment:
Can we change this to be of a union that has the list of types we allow. e.g 

```
display: 'block' | 'inline-block' | 'flex';
```

Reply:
Ah yep, this would be better, thanks!

Reply:
This has been updated.